mokapot/mokapot.py

```diff
@@ -82,13 +83,24 @@ def main():
                 dataset.add_proteins(proteins)
 
     # Define a model:
-    model = PercolatorModel(
-        train_fdr=config.train_fdr,
-        max_iter=config.max_iter,
-        direction=config.direction,
-        override=config.override,
-        subset_max_train=config.subset_max_train,
-    )
+    if config.init_weights:
+        model_files = os.listdir(str(config.init_weights))
```

Comment:
This will change with my proposed change to the command line argument, but in general I prefer that `pathlib.Path` functionality be used over `os` for path operations.